github-notifier.el

```diff
@@ -185,9 +189,15 @@ will return an API."
 
 (defun github-notifier-visit-github ()
   (interactive)
-  (browse-url (github-notifier-get-url "/notifications")))
+  (browse-url (github-notifier-get-url "/notifications"))
+  (setq github-notifier-last-notification-checked (format-time-string "%FT%TZ" (current-time) t))
+  (force-mode-line-update t))
+
+(defun github-notifier-notifications-checked ()
+  (and github-notifier-unread-count (> github-notifier-unread-count 0)
+       github-notifier-last-notification github-notifier-last-notification-checked
+       (string< github-notifier-last-notification github-notifier-last-notification-checked)))
```

Comment:
Yeah I toyed around with time conversion but reached the same conclusion: more code.